backend/tests/Feature/PublicationTest.php

```diff
@@ -20,4 +24,97 @@ public function testPublicationsCanBeCreatedWithCustomNamesThatAreNotDuplicates(
         $this->expectException(QueryException::class);
         Publication::factory()->create(['name' => 'Custom Name']);
     }
+
+    public function testPublicationsCanBeCreatedViaMutationByAnApplicationAdministrator()
+    {
+        $user = User::factory()->create();
+        $user->assignRole(Role::APPLICATION_ADMINISTRATOR);
+        $this->actingAs($user);
+        $response = $this->graphQL(
+            'mutation CreatePublication {
+                createPublication(publication:{name:"Custom Publication for Unit Testing"}) {
+                    name
+                }
+            }'
+        );
+        $expected_data = [
+            'createPublication' => [
+                'name' => 'Custom Publication for Unit Testing',
```

Comment:
Create a publication with leading and trailing whitespaces to test the trim directive.